skimage/transform/tests/test_warps.py

```diff
@@ -620,27 +646,33 @@ def test_log_polar_mapping():
     assert np.allclose(coords, ground_truth)
 
 
-def test_linear_warp_polar():
+@testing.parametrize('dtype', [np.float32, np.float64])
+def test_linear_warp_polar(dtype):
     radii = [5, 10, 15, 20]
     image = np.zeros([51, 51])
     for rad in radii:
         rr, cc, val = circle_perimeter_aa(25, 25, rad)
         image[rr, cc] = val
+    image = image.astype(dtype, copy=False)
     warped = warp_polar(image, radius=25)
+    assert warped.dtype == _supported_float_type(dtype)
     profile = warped.mean(axis=0)
     peaks = peak_local_max(profile)
     assert np.alltrue([peak in radii for peak in peaks])
 
 
-def test_log_warp_polar():
+@testing.parametrize('dtype', [np.float32, np.float64])
```

Comment:
And here.

Reply:
Thanks! I have added float16 to the cases you pointed out.